.github/workflows/flatpak.yml

```diff
@@ -120,39 +31,33 @@ jobs:
         sed -i 's/<version>/${{ github.event.release.tag_name }}/g' $GITHUB_WORKSPACE/share/org.getmonero.Monero.metainfo.xml
         sed -i 's/<date>/'"$(date '+%F')"'/g' $GITHUB_WORKSPACE/share/org.getmonero.Monero.metainfo.xml
 
-    - name: Install deps
-      run: dnf -y install docker
-
-    - name: Setup QEMU
-      uses: docker/setup-qemu-action@v2
-      with:
-        platforms: arm64
-
-    - name: Restore flatpak-builder
-      uses: actions/download-artifact@v4.1.7
-      with:
-        name: flatpak-builder-${{ matrix.arch }}
-
-    - name: Untar flatpak-builder
-      run: tar -xvf flatpak-builder.tar
+    - name: Validate Flatpak manifest
+      run: flatpak-builder-lint manifest share/org.getmonero.Monero.yaml
 
     - name: Build flatpak
-      uses: flatpak/flatpak-github-actions/flatpak-builder@v6
+      uses: flathub-infra/flatpak-github-actions/flatpak-builder@821846071bc503ec98c1db6b75530b571606e1a5
```

Comment:
why the change to flathub-infra? the example here still uses flatpak/.../@v6 https://github.com/flatpak/flatpak-github-actions?tab=readme-ov-file#multi-arch-build-using-public-arm64-runners

Reply:
> why the change to flathub-infra? the example here still uses flatpak/.../@v6 https://github.com/flatpak/flatpak-github-actions?tab=readme-ov-file#multi-arch-build-using-public-arm64-runners

I don't remember well why I used it then, there were some reasons, but [not anymore](https://github.com/flathub-infra/flatpak-github-actions/issues/17)